BREEZE.rst

```diff
@@ -2106,13 +2106,30 @@ Those are all available flags of ``generate-constraints`` command:
 
 In case someone modifies setup.py, the scheduled CI Tests automatically upgrades and
 pushes changes to the constraint files, however you can also perform test run of this locally using
-the procedure described in `Refreshing CI Cache <dev/REFRESHING_CI_CACHE.md#manually-generating-constraint-files>`_
+the procedure described in the
+`Manually generating image cache and constraints <dev/MANUALLY_GENERATING_IMAGE_CACHE_AND_CONSTRAINTS.md>`_
 which utilises multiple processors on your local machine to generate such constraints faster.
 
 This bumps the constraint files to latest versions and stores hash of setup.py. The generated constraint
 and setup.py hash files are stored in the ``files`` folder and while generating the constraints diff
 of changes vs the previous constraint files is printed.
 
+Updating constraints
+""""""""""""""""""""
+
+Sometimes (very rarely) we might want to update individual packages in constraints that we generated and
+tagged already in the past. This can be done using ``breeze release-management update-constraints`` command.
+
+These are all available flags of ``update-constraints`` command:
+
+.. image:: ./images/breeze/output_release-management_update-constraints.svg
+  :target: https://raw.githubusercontent.com/apache/airflow/main/images/breeze/output_release-management_update-constraints.svg
+  :width: 100%
+  :alt: Breeze update-constraints
+
+You can read more details about what happens when you update constraints in the
+`Manually generating image cache and constraints <dev/MANUALLY_GENERATING_IMAGE_CACHE_AND_CONSTRAINTS.md>`_
```

Comment:
Not really. Those are separate processes that are very rarely /exceptionally used - and having a separate document for them is better.

Reply:
Makes sense. Thanks!

Reply:
Oh wait, maybe a misunderstanding. I meant instead of including = `<dev/MANUALLY_GENERATING_IMAGE_CACHE_AND_CONSTRAINTS.md>` to show the options, should we have it instead in "text" like we do for other commands?

Reply:
Not sure if I follow? 

We are not including it - we include the link to the docs - what else should we do here you think ?

Reply:
example?

Reply:
Was wondering if we can have a section like this https://github.com/apache/airflow/blob/main/BREEZE.rst?plain=1#L2033-L2060

Where we explain the command options. But looks like we already do quite a bit here: https://github.com/apache/airflow/pull/33144/files#diff-48b19ec509d2b387eabfa3066d32a76a26bcea94ab173fe245ad5c497dc43654R317-R334

Go ahead and take the call here.

Reply:
I think for those "obscure" commands that likely only I will ever do (or my direct successor or someone doing an emergency fix in my absence) - having those detail in separate "exceptional process" description is way better - because I can provide a context there, and if there is really a need to fix things like mentioned there, you need to deeply understand what you are doing, so just showing the command and redirecting there - to the place where you can at least have a chance to get context, is much better.